github/orgs_teams.go

```diff
@@ -226,14 +214,15 @@ func (s *OrganizationsService) ListTeamRepos(team int, opt *ListOptions) ([]Repo
 // permissions team has for that repo.
 //
 // GitHub API docs: http://developer.github.com/v3/orgs/teams/#get-team-repo
```

Comment:
This might be out of scope for this PR, but this link http://developer.github.com/v3/orgs/teams/#get-team-repo doesn't seem to be valid anymore. Should it be updated to something else, or replaced by the new one you added on the following line?